### ISSUE TITLE
Map Centre/GPS position switch is very hard to move

### ISSUE THREAD

**Issue by AndrewM-** [geopaparazzi/geopaparazzi#637]
If I want to change the position of the switch, I usually have to try several times.  Is it locked and only unlocked by a long press or something?  Gpap 5.6.5

**Comment by AndrewM-** [comment]
I have worked out that it just needs to be tapped to change it.

**Comment by AndrewM-** [comment]
I have tested this issue some more.  The control used is actually a button with a graphical representation as a switch or slider, so it did not occur to me just to tap the control.  I might be advanced with Windows but an still clumsy with Android.  As my finger hides the control when I am activating it, it tool a long time to figure out what was actually happening.

I found the hot spot to be very small and hard to hit.  Even though I know how the control works now, I can get it to work only about 50% of the time.  In contrast, the same type of controls in the Android settings seem to be a little easier to use.  They seem to have a wider hot spot.  Could the labels also be made into links that flick the switch.  The labels are much larger.  Perhaps the label could be bolded when the option it describes is active.  

A lot of the work I do is vehicle based in places with bad roads so I need something that is fast to use  when being shaken around.
